src/ForceTorqueSensor.cc

```diff
@@ -247,7 +250,7 @@ bool ForceTorqueSensor::Update(const std::chrono::steady_clock::duration &_now)
   applyNoise(TORQUE_Y_NOISE_N_M, measuredTorque.Y());
   applyNoise(TORQUE_Z_NOISE_N_M, measuredTorque.Z());
 
-  msgs::Wrench msg;
+  msgs::Wrench &msg = this->dataPtr->measuredWrench;
   *msg.mutable_header()->mutable_stamp() = msgs::Convert(_now);
   auto frame = msg.mutable_header()->add_data();
```

Comment:
there is an issue here; working on a fix

Reply:
it was adding a new key-value pair for the `"frame_id"` to the header map at every timestep. I've changed it in https://github.com/gazebosim/gz-sensors/pull/449/commits/738a9da22d3d74cc34d70d69e329f1e4e3faa17e to allocate the key-value pair in the constructor and update it at each timestep instead